src/dmd/expressionsem.d

```diff
@@ -11707,6 +11722,74 @@ Lerr:
     return errorExp();
 }
 
+/***************************************
+ * If expression is shared, check that we can access it.
+ * Give error message if not.
+ * Params:
+ *      e = expression to check
+ *      sc = context
+ * Returns:
+ *      true on error
+ */
+bool checkSharedAccess(Expression e, Scope* sc)
+{
+    if (!global.params.noSharedAccess ||
+        sc.intypeof ||
+        sc.flags & SCOPE.ctfe)
```

Comment:
This line and the previous one should be doubly indented lest there is confusion with the body of the `if` statement.

Reply:
added `{ ... }` for that.